test/mocha_mongodb.js

```diff
@@ -27,7 +27,9 @@ module.exports = {
     'test/integration/node-specific/examples/handler.test.js',
     'test/integration/node-specific/examples/aws_handler.js',
     'test/integration/node-specific/examples/aws_handler.test.js',
-    'test/integration/node-specific/examples/setup.js'
+    'test/integration/node-specific/examples/setup.js',
+    'test/integration/node-specific/examples/transactions.test.js',
```

Comment:
This is needed to utilize `examples/setup.js`, which makes it possible to import mongodb as `const { MongoClient } = require('mongodb');` inside the example files (see my comment above).